src/components/composite/sections/ContextSectionIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ComponentType, FC } from 'react';
```

Comment:
How is this change related to the story?

Reply:
Please see https://github.com/alkem-io/client-web/pull/1973#discussion_r849360235